config/initializers/hydra_editor.rb

```diff
@@ -0,0 +1,5 @@
+HydraEditor.models = ["GenericFile"]
+HydraEditor::Fields::Generator.factory = ScholarsArchive::Fields::InputFactory.new(
+  HydraEditor::Fields::Factory,
+  DecoratorList.new(HasHintOption, HasURIInputType)
```

Comment:
If we decide to not use the HasHintOption quite yet, we can remove it from here as well.